src/screens/augmentedReality/ARShowScreen.js

```diff
@@ -180,6 +209,11 @@ var styles = StyleSheet.create({
     position: 'absolute',
     zIndex: 1
   },
+  screenRecording: {
+    bottom: 200,
+    padding: 15,
+    right: 10
```

Comment:
how does it look like now?

Reply:
|iPhone 13 Pro Max|iPhone 7|
|--|--|
![IMG_C77827D2D412-1](https://user-images.githubusercontent.com/11755668/179163792-a1ed33c5-c561-4c64-bf9e-cad873ae081e.jpeg) | ![IMG_60D2BADF999A-1](https://user-images.githubusercontent.com/11755668/179163814-41da5b9e-e404-4402-bcb0-1cf322bdeae9.jpeg)

Reply:
why is the record button smaller?
and could we add some transparency to the background gray, like in the figmas?

Reply:
The record button is not on the Figma. I added it myself. I thought it would be a bit confusing if the two buttons were the same size. But I can make it the same size.

 I will add transparency.

Reply:
![IMG_EBC1EF28AFAA-1](https://user-images.githubusercontent.com/11755668/179208941-2839294a-295e-4f10-990e-5dc711571484.jpeg)

https://github.com/ikuseiGmbH/smart-village-app-app/pull/577/commits/9d0ff4222b5d3ffc0209a9ac08b4293757584734

Reply:
> I thought it would be a bit confusing if the two buttons were the same size.

but with the x on the top right it would be three different sizes which i think is most confusing. now it looks good.

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/577/commits/0e3b8d076de6807d14cb8ba6734ed61148f5e196